bridge_adaptivity/module/static/module/js/module.js

```diff
@@ -360,9 +360,12 @@
         const modalSuccessListener = (url) => {
             return result => {
                 let modal = $('#modal-wrapper');
-                if (result.status !== undefined && result.status === 'ok' && result.collection_slug !== undefined) {
-                    var new_url = new URL(window.location.href);
+                var new_url = new URL(window.location.href);
+                 // FixMe: This check  is needed to remove in another place
```

Comment:
Please rewrite the comment id doesn't clear enough.

Reply:
Done